src/git/cli.ts

```diff
@@ -29,7 +26,7 @@ function _execute(cwd: string, subcommand: string, options: string[] = []) {
 
```

Comment:
As far as I can see, for the array in `QUOTE_PATH` to help with escaping you would need to use [`execa`'s array syntax](https://github.com/sindresorhus/execa/blob/main/docs/escaping.md#array-syntax) (which is the concept I was referring to in the issue)

Reply:
Oh right. I did a standalone test using array and nesting it.

But forgot to do it here, I guess on line 28 for command as array rather than strings.

Reply:
Yes, (besides `git` needing to be extra). Line 29 could look like this as far as I see it:

```ts
return execa("git", [...QUOTE_PATH, subcommand, ...options] ,{ cwd });
```

Reply:
thanks. I added #116 as a task to update to module type and then this PR will be possible